tox.ini

```diff
@@ -61,6 +61,8 @@ deps =
     bddtests: -r requirements/bddtests.txt
     dockercompose: -r requirements/dockercompose.txt
     {env:EXTRA_DEPS:}
+depends =
+    coverage: tests
```

Comment:
Tells tox not to start `coverage` until `tests` has finished. Only takes effect if both `coverage` and `tests` are included in the envlist (e.g. `tox --parallel -e coverage,tests,...`)